docs/cli/cmd-options.mdx

```diff
@@ -656,6 +672,24 @@ Specify a delay before the workflow starts.
 
 Backfill start time.
 
+## target-namespace
+
+Namespace in which a handler worker will be polling for Nexus tasks on.
+
+## target-task-queue
+
+Task Queue in which a handler worker will be polling for Nexus tasks on.
+
+## target-url
+
+URL to direct Nexus requests to.
```

Comment:
Is this the internal URL versus the Nexus endpoint? Maybe be clearer about that? Is it a Namespace endpoint? Is it one that routes requests to the Namespaces that handle the requested work? What kind of URL is it?